config.py

```diff
@@ -103,11 +104,11 @@
 
 # ``l_freq``  : the low-frequency cut-off in the highpass filtering step.
 # Keep it None if no highpass filtering should be applied.
-l_freq = 1
+l_freq = 0.01
 
 # ``h_freq``  : the high-frequency cut-off in the lowpass filtering step.
 # Keep it None if no lowpass filtering should be applied.
-h_freq = 45
+h_freq = 100
 
 
 ###############################################################################
```

Comment:
@SophieHerbst I get an error when I try to run the 02-apply_maxwell_filter.py: 
  File "C:\Users\Dragana\Anaconda3\lib\site-packages\mne\preprocessing\maxwell.py", line 436, in maxwell_filter
    if read_lims[-1] != end:

IndexError: index -1 is out of bounds for axis 0 with size 0

Reply:
Hi Dragana,
I vaguely remember having had that problem with an MEG file for which I had switched the recording on and off, causing discontinuities. I don't remember what I did though..
Can you try to visualize the data using the first script and look whether there are any boundaries (marked by red lines)? If they appear largely before your first triggers, I would crop the file manually and save it again. 
I am not an NS tomorrow, but available in the morning. 
Sophie

Reply:
Hello Sophie, We clicked twice on 2 occasions on Record raw and there are 2 files per run. But one of them is much less memory heavy. I proceeded to analyze the 'good' runs, and will cut the one which has the boundaries.